src/utils/cues.js

```diff
@@ -49,7 +49,7 @@ export function newCue (track, startTime, endTime, captionScreen) {
 
       cue.align = 'left';
       // Clamp the position between 0 and 100 - if out of these bounds, Firefox throws an exception and captions break
-      cue.position = Math.max(0, Math.min(100, 100 * (indent / 32) + (navigator.userAgent.match(/Firefox\//) ? 50 : 0)));
+      cue.position = Math.max(0, Math.min(100, 100 * (indent / 32)));
```

Comment:
I remember this had to be left in for captions to position correctly on firefox, but if this was taken out I bet it's all good now.
There is also some code in the player control bar that shifts them back, will have to make sure that's tested